.github/workflows/calibration.yaml

```
@@ -163,6 +163,13 @@ jobs:
           mkdir -p "${task_output_dir}"
 
           python3 "${{ needs.setup.outputs.worktree_path }}/src/scripts/calibration_analyses/analysis_scripts/process.py" "${task_output_dir}" "${RUN_NAME}" "${{ needs.setup.outputs.worktree_path }}/resources"
+
+          # Print info.txt file with some useful information
+          info_txt="${task_output_dir}/info.txt"
```

I think this would put the file in the `022_long_run_all_diseases_process` directory. Can we put it in the parent (I guess `${{ needs.setup.outputs.output_dir }`)?

---

You're right :facepalm: 
```suggestion
          info_txt="${{ needs.setup.outputs.output_dir }}/info.txt"
```